packages/payload/src/fields/config/client.ts

```diff
@@ -126,6 +126,13 @@ export const createClientBlocks = ({
       clientBlock.jsx = jsxResolved
     }
 
+    if (!clientBlock?.admin?.disableBlockName) {
```

Comment:
Does this get defaulted to true somewhere that I am missing? If not this is a bit confusing no?

Reply:
I think I just got confused since the boolean flipped in the new form. I simplified it now